AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift

```diff
@@ -12,6 +12,16 @@ import Foundation
 @objcMembers public class LocalVideoConfiguration: NSObject {
 
     /// The flag to disable/enable simulcast, default to true
-    /// only for localVideo, not work for content share
-    public var simulcastEnabled: Bool = true
+    /// For local video use only, will not work for content share
+    public var simulcastEnabled: Bool
```

Comment:
why is this removed?

Reply:
It is refactored into the constructor.

Reply:
ok. Just to note, it won't work on objective-C since it doesn't support default parameters.

Reply:
discussed offline. will merge.